tracer/src/Datadog.Trace/Iast/Aspects/System.Net/HttpClientAspect.cs

```diff
@@ -54,6 +54,7 @@ public class HttpClientAspect
     public static string Review(string parameter)
     {
         IastModule.OnSSRF(parameter);
+        RaspModule.OnSSRF(parameter);
```

Comment:
Duplicating these calls in all the sinks seems inefficient. I'd put them in an aggregator class.

Reply:
I could do that in a separate PR, if it's fine, since that would involve changing also some other non SSRF aspects. Thanks!